openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/raw-wb-board.js

```diff
@@ -369,6 +370,52 @@ var Wb = function() {
 					minHeight: 140
 					, minWidth: 255
 				});
+				tools.find('.om-icon.dweet').click(function(){					
+			        dweet = OmUtil.tmpl("#wb-dweet");			        
+			       	dweet[0].id='wb-dweet' + dweetIDCount++;        
+			        dweet[0].style.bottom = '100px';
+			        dweet[0].style.position= 'absolute';
+			        dweet[0].style[(Settings.isRtl ? 'left' : 'right')] = '100px';        			        					      	         
```

Comment:
this `Settings.isRtl ? 'left' : 'right'` code is obsolete
alignment will be changed automatically in case of RTL

Reply:
Done